### DISCUSSION TITLE
Use color channels for phase and stereo image

### DISCUSSION THREAD

**Discussion by 1R053** [riffusion/riffusion#8] [Ideas]
Hi,

has anyone ever tried to use the 3 color channels a bit differently, e.g.:
R: Volume for the frequency (mono signal)
G: Phase for the frequency (mono signal)
B: stereo volume difference between left and right signal, so any resulting output of the mono frequency and phase can simply be volume shifted

last one is not perfect, as certainly a phase shifting for stereo also would be nice, but maybe its enough...

**Comment by hmartiro** [comment]
Phase is hard by itself, it's very chaotic. But I've heard some ideas around phase velocity. My audio knowledge is fairly limited so welcome any experimentation. Supporting stereo with color channels I think is more straightforward and I think would work.

There's some related discussion here: https://www.reddit.com/r/riffusion/comments/zn1sw1/expand_frequency_fidelity_using_rgb_frequency/
